app/(pages)/evidence/page.tsx

```diff
@@ -1,7 +1,22 @@
 import { FC } from 'react';
 
+import "./evidence.scss"
+import Form from '@/app/components/Evidence/form';
+import SideImage from '@/app/components/Evidence/sideImage';
```

Comment:
Please check how components are exported on other files and follow the same pattern